### DISCUSSION TITLE
Siguiente paso

### DISCUSSION THREAD

**Discussion by GloriaCoralCerecedo** [LaunchX-InnovaccionVirtual/FrontEnd-Mision#529] [Q&A]
Hola, entonces para que me quede más claro es que me distraje un poco en el live por una situación.

El martes nos darán un live para hablarnos y ayudarnos con el estrés o algo así

Y el jueves ya va a ser un live que hablara sobre el Back End

Lo cual quiere decir que para los que estén atrasados y les faltó algo se pueden poner al corriente (me falta la práctica de VUE)

Y lo del portafolio no escuché bien se que es lo que estábamos haciendo, supongo que habrá que reenviarlo o algo así. 

Disculpen que escriba tanto jajaa, es solo para asegurarme

**Comment by OmarVenturaP** [comment]
Exactamente eso es, tal cual lo mencionas. Tu repo se puede decir que es tu portafolio.

**Reply by GloriaCoralCerecedo** [reply]
Aunque te lo varios repositorios  no solo uno

**Comment by krlosbyte** [comment]
Supongo que es sobre el live en Instagram, lo hubieras grabado, yo estaba esperando y se me olvido, entre 17 minutos tarde y según ya se había acabado

**Reply by GloriaCoralCerecedo** [reply]
Creo que lo iban a subir al teams, creo

**Reply by chistoperez** [reply]
https://teams.microsoft.com/l/meetup-join/19%3ameeting_ZTFjZDk5YWItYzE4Mi00Yjk2LTlkOGItNDA3OWIyZmJhNDQ3%40thread.v2/0?context=%7b%22Tid%22%3a%224ae54b05-b77e-4224-aef1-8661422e0816%22%2c%22Oid%22%3a%22b27891aa-c2b1-4e44-85dc-357f2a31b12b%22%2c%22IsBroadcastMeeting%22%3atrue%7d

**Reply by GloriaCoralCerecedo** [reply]
Gracias!

**Comment by JuanSalvador73** [comment]
Gracias por la info :D
